scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -176,6 +219,18 @@ def validate_file(file):
     if not os.path.exists(file):
         raise MotrError(errno.ENOENT, f"{file} does not exist")
 
+def validate_files(files):
+    for file in files:
+        if not os.path.exists(file):
+            raise MotrError(errno.ENOENT, f"{file} does not exist")
+
+def create_dirs(self, dirs):
+    for dir in dirs:
+        if not os.path.exists(dir):
+            print(f" On line 230 {dir}")
```

Comment:
on line 230 should be removed.
add a message creating directory

Reply:
Done ed4d648